src/grpc/helium_state_channels_impl.erl

```diff
@@ -150,11 +150,26 @@ is_active_sc(
 ) ->
     lager:info("executing RPC is_active with msg ~p", [_Message]),
     {ok, CurHeight} = get_height(),
-    Response0 = #gateway_sc_is_active_resp_v1_pb{
-        active = check_is_active_sc(SCID, SCOwner, Chain),
-        sc_id = SCID,
-        sc_owner = SCOwner
-    },
+    Ledger = blockchain:ledger(Chain),
+    Response0 =
+        case get_ledger_state_channel(SCID, SCOwner, Ledger) of
+            {ok, Mod, SC} ->
+                #gateway_sc_is_active_resp_v1_pb{
+                    active = true,
+                    sc_id = SCID,
+                    sc_owner = SCOwner,
+                    sc_expiry_at_block = Mod:expire_at_block(SC),
+                    sc_original_dc_amount = get_sc_original(Mod, SC)
+                };
+            _ ->
+                #gateway_sc_is_active_resp_v1_pb{
+                    active = false,
+                    sc_id = SCID,
+                    sc_owner = SCOwner,
+                    sc_expiry_at_block = 0,
```

Comment:
should these not be `undefined` to meet the proto spec and avoid serializing a 0?

Reply:
yeah might be better

Reply:
fixed